test/ScanDependencies/scan_return_module_version_textual_interface.swift

```diff
@@ -0,0 +1,23 @@
+// REQUIRES: objc_interop
+// RUN: %empty-directory(%t)
+// RUN: %empty-directory(%t/clang-module-cache)
+// RUN: %empty-directory(%t/DependencyModules)
+
+// Emit a textual module dependency
```

Comment:
It looks to me like this will actually only emit a binary module dependency, right? Which means `-module-load-mode prefer-interface` below is also possibly unnecessary?

Reply:
ah, right. I guess it's there because I've copied this command from a similar test. Will remove.